tests/test_queues_stat.py

```diff
@@ -44,6 +44,51 @@ def test_getQueueCounters(self):
             self.assertEqual(str(value0.name), str(oid))
             self.assertEqual(value0.data, 1)
 
```

Comment:
can you add a similar test for multi-asic as well:
tests/namespace/test_queues_stat.py